### PR TITLE
processing status

### PR DESCRIPTION
**Pull request by Bento007**
#### Reviewers
**Functional:** @MDunitz 

**Readability:** @tihuan 

---
This PR is blocked on #853 and #866
## Changes
- add updates to processing_status from the upload container.

### CODE REVIEW

File: backend/config/corpora-api.yml [hunk 1]
```diff
@@ -329,6 +329,10 @@ paths:
                 properties:
                   dataset_uuid:
                     $ref: "#/components/schemas/dataset_uuid"
+                  processing_status:
+                    type: string
+                    enum:
+                      [ PENDING, SUCCESS,FAILURE]
```
Comment (tihuan) [inline backend/config/corpora-api.yml:335]: Created a PR to fix this and add prettier to format `yaml`! https://github.com/chanzuckerberg/corpora-data-portal/pull/869

PTAL thank you :D

**Comment by tihuan** [comment]
@Bento007 looks like this PR is blocked by `codecov` 🙏 PTAL thank you!

**Review by MDunitz — APPROVED** [review]
lgtm -- is this something we could add functional or integration tests for?

**Comment by Bento007** [comment]
@MDunitz yes we should

**Pull request merged**